src/components/Forms.js

```diff
@@ -33,14 +33,16 @@ export const TextArea = props => {
 };
 
 export const Input = props => {
-  
+
   const {
     className,
     onChange,
     placeholder,
     readOnly,
     value,
     icon,
+    innerRef,
+    type
```

Comment:
Comma dangles

We should enforce this via eslint

I have a branch going

Reply:
I added trailing commas to my prettier settings and got it so format the file on-save, this should help in the meantime